StreamChat.podspec

```diff
@@ -9,15 +9,19 @@ Pod::Spec.new do |spec|
   spec.author = { "getstream.io" => "support@getstream.io" }
   spec.social_media_url = "https://getstream.io"
   spec.swift_version = "5.2"
-  spec.platform = :ios, "11.0"
+
+  spec.ios.deployment_target  = '11.0'
+  spec.osx.deployment_target  = '10.15'
```

Comment:
We would need to bump `spec.version` to activate changes. Otherwise `osx` is ignored.